front/src/Api/iframe/ui.ts

```diff
@@ -2,13 +2,17 @@ import { isButtonClickedEvent } from '../Events/ButtonClickedEvent';
 import type { ClosePopupEvent } from '../Events/ClosePopupEvent';
 import { IframeApiContribution, sendToWorkadventure } from './IframeApiContribution';
 import { apiCallback } from "./registeredCallbacks";
-import {Popup} from "./Ui/Popup";
-import type {ButtonClickedCallback, ButtonDescriptor} from "./Ui/ButtonDescriptor";
+import { Popup } from "./Ui/Popup";
+import type { ButtonClickedCallback, ButtonDescriptor } from "./Ui/ButtonDescriptor";
+import { TriggerMessage } from './Ui/TriggerMessage';
+import { isMessageReferenceEvent } from '../Events/TriggerMessageEvent';
 
 let popupId = 0;
 const popups: Map<number, Popup> = new Map<number, Popup>();
 const popupCallbacks: Map<number, Map<number, ButtonClickedCallback>> = new Map<number, Map<number, ButtonClickedCallback>>();
 
+const triggerMessages: Map<string, TriggerMessage> = new Map<string, TriggerMessage>();
```

Comment:
You are appending in `triggerMessages` new messages, but we are never removing messages from this Map.
I know the messages have a quite small memory footprint, but still, it could be cool to clean them when they are closed.

Also, I don't think we can have 2 messages open at the same time. So maybe, rather than using a Map here, we could simply have a single reference to a message? (and if a 2nd message is open, we close the first one?)